pages/genes/[id].js

```diff
@@ -110,3 +110,19 @@ export const getServerSideProps = async ({ params, req }) => {
   }
   return errorObjectToProps(gene);
 };
+
+const EnsemblLink = ({ geneid, taxa }) => {
+  const organism = taxa.replace(/ /g, "_");
+  return (
+    <a href={`http://www.ensembl.org/${organism}/Gene/Summary?g=${geneid}`}>
+      {geneid}
+    </a>
+  );
+};
+
+EnsemblLink.propTypes = {
+  // GeneID to display as a link
+  geneid: PropTypes.string.isRequired,
+  // Metadata that affects certain dbxrefs
+  taxa: PropTypes.string.isRequired,
+};
```

Comment:
This component looks good! Could you move it above the `Gene` component? `getServerSideProps` is server-only code (NextJS looks for that function name specifically and removes it from browser code) and I’d like to keep server-only code at the bottom of files.

Reply:
I'll move the component to the top. Also, we currently don't have any ENTREZ dbxrefs in alpha database but will have them soon after we figure out a new gene loading script (wrangling).